### DISCUSSION TITLE
V1.35

### DISCUSSION THREAD

**Discussion by Unlucky-Life** [Unlucky-Life/ankimon#242] [Show and tell]
- Fixed Multiple Attacks for one Pokemon issue
- Added Paginator to pokemon collection window do not have crashes
- added logger to see all messages
- add shinies
- reworked spawn rates
- added individual id, catch date, defeated pokemon amount, friendship score
- code was completely refactored into a "object-oriented" style
- added ankimon tracker, to check for debugging issues
- fixed pokemon evolution by level
- fixed pokemon evolution by item
- added daily item shop
- added trainer card
- added leveling of a trainer

<hr /><em>This discussion was created from the release <a href='https://github.com/Unlucky-Life/ankimon/releases/tag/main-a685e882'>V1.35</a>.</em>

**Comment by OuraN2O** [comment]
Hey, I find the formulation a bit confusing, maybe you should just copy what the dialog is when forcing a sync in anki ?
![image](https://github.com/user-attachments/assets/f2f38c6c-612e-43d8-9d37-89ad48b6c4f8)
Even I needed to think multiple times about which button to press to not nuke my account lol

**Reply by OuraN2O** [reply]
Also, I chose a starter pokemon when first installing the addon as prompted, then I tried to retrieve the anwkiweb data I had synced but on the next restart it asked me again and it was pointing that my last starter (the one I had synced in ankiweb) was unknown, so I loaded my new starter to ankiweb and it overwritten my entire collection with my new starter.
Not a big deal for me as I still have my files, but for some people it might be and it is confusing that you get prompted for a new starter before asked between local and ankiweb

**Reply by Unlucky-Life** [reply]
@OuraN2O Thats true havent addapted the code for the first start of ankimon. Still need to adapt those functions.
What would you suggest renaming the buttons ? Or what exactly would you show in the dialog ?

**Reply by OuraN2O** [reply]
I think renaming the 2 buttons should be enough to eliminate confusion, for example "Export data to ankiweb" and "Import data from ankiweb"

**Comment by OuraN2O** [comment]
I think that the config explanation needs some rewriting. It should follow the order of what is in the config file to be able to read it through and not having to search what a setting does. Also, I didn't find an explanation for some of them, maybe was it not changed to accommodate the changes of this (pre-)release ? And last but not least, there's no markdown so it's tricky to differentiate the setting itself from the explanation of the setting 
![image](https://github.com/user-attachments/assets/577510d6-dd89-449b-b26f-e5a481948b58)

**Reply by Unlucky-Life** [reply]
@OuraN2O true some configurations werent changed => Could you list the missing configuration explanations ?
(The descriptions for the settings are now in a seperate json file): 
https://github.com/Unlucky-Life/ankimon/blob/main/src/Ankimon/lang/setting_description.json

**Reply by OuraN2O** [reply]
![image](https://github.com/user-attachments/assets/94d17e54-1328-4b1a-81d9-593291a61438)
Those in blue were found, and in red are the ones missing. Also, how about the markdown and the order ?

**Reply by Unlucky-Life** [reply]
ah ok, we will remove this settings window though :) !
